docs/tutorials/getting-started-eclipse.md

```diff
@@ -0,0 +1,71 @@
+---
+type: tutorial
+layout: tutorial
+title:  "Getting Started with Eclipse Luna"
+description: "This tutorial guides through installing Kotlin plugin for Eclipse and creating Hello World application."
+authors: Nikolay Krasko
+date: 2015-03-28
+showAuthorInfo: true
+---
+### Setting up the environment
+We will show how to install Kotlin plugin in Eclipse Luna SR2 (4.4.2) and create a simple Hello World application.
```

Comment:
Don't think we need to repeat description here. I'd just mention version of Eclipse required (if necessary)